crates/services/txpool_v2/src/service.rs

```diff
@@ -382,8 +400,9 @@ where
         let time_txs_submitted = self.pruner.time_txs_submitted.clone();
         let tx_id = transaction.id(&self.chain_id);
         let utxo_validation = self.utxo_validation;
+        let metrics = self.metrics;
 
-        move || {
+        let op = move || {
```

Comment:
nit: Can we name this something more descriptive of what it's doing?

Reply:
It should now be `insert_transaction_thread_pool_op`: 889b92dd09